scipy/spatial/qhull.pyx

```diff
@@ -1009,10 +1007,8 @@ cdef void _visit_voronoi(qhT *_qh, void *ptr, vertexT *vertex, vertexT *vertexA,
     p[2*qh._nridges + 1] = point_2
 
     # Record which voronoi vertices constitute the ridge
-    cur_vertices = []
-    for i in xrange(qh_setsize(_qh, centers)):
-        ix = (<facetT*>centers.e[i].p).visitid - 1
-        cur_vertices.append(ix)
+    cur_vertices = [(<facetT*>centers.e[i].p).visitid - 1
+                    for i in xrange(qh_setsize(_qh, centers))]
```

Comment:
There's test failures on TravisCI for `voronoi` functionality that looks real, due to this change or the one above. Can you debug or revert?

Reply:
Yes, this like was breaking tests. I'm not sure what was wrong in the above change, so I reverted it.